### DISCUSSION TITLE
IDE Font Size

### DISCUSSION THREAD

**Discussion by resih5** [apache/netbeans#7805] [Ideas]
I know this is a minor thing to most users, but is there a way to increase IDE font size in NetBeans version 23? Not the editor font size, but rather the font size of the IDE menu, files list, tabs, etc... In older versions we could edit the netbeans.conf file and add --font size to end of the default options. I tried this with new version and it does not work for me.

**Comment by matthiasblaesing** [comment]
Tested with a build based on current master and font size change still works. You might have a typo. Tested with:

```
bin/netbeans-dev/bin/netbeans --fontsize 20
bin/netbeans-dev/bin/netbeans --fontsize 8
```

and also via conf:

![image](https://github.com/user-attachments/assets/44ace151-bf83-4e22-934d-c5e7ccaaef5c)

resulting in:

![image](https://github.com/user-attachments/assets/126685ea-14d2-4012-b402-d2a1130342e0)

**Reply by resih5** [reply]
I swear I tried to change the config file 2-3 times and it did not work, but now, it works better than it ever has. Thank you for the help

**Comment by neilcsmith-net** [comment]
Incidentally, there is a second way to achieve this, which is useful with installers where editing the conf file is awkward (or on macOS will just break).

Go to `Tools / Options / Appearance / FlatLaf` and click on the `Edit custom properties` button.  This will open a properties file in the editor.  Add ..

```properties
defaultFont=24
```

.. save the file, and restart the IDE.  You can override any look and feel property in that file, incidentally.  You can also change more than just the font size - see eg. https://www.formdev.com/flatlaf/properties-files/#font  Links to the documentation are in the header of the file that opens.
